_includes/head.html

```diff
@@ -50,4 +50,8 @@
     ga('create', 'UA-73726852-1', 'auto');
     ga('send', 'pageview');
   </script>
+  <script>
+    var github_repo = "{{ site.repository | escape }}";
```

Comment:
github.js is ostensibly a static resource, so I don't think it should make use of jekyll tags (especially `github_path`, since that is page-specific). I can stick the script tag in doc.html though.